docs/root/extending/extending.rst

```diff
@@ -19,8 +19,11 @@ types including:
 * :ref:`Stat sinks <arch_overview_statistics>`
 * :ref:`Tracers <arch_overview_tracing>`
 * :ref:`Request ID <arch_overview_tracing>`
-* Transport sockets
-* BoringSSL private key methods
+* :ref:`Transport sockets <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CommonTlsContext.CertificateProvider.typed_config>`
+* :ref:`BoringSSL FIPS <arch_overview_ssl_fips>`
+* :ref:`Watchdog action <envoy_v3_api_msg_config.bootstrap.v3.Watchdog.WatchdogAction>`
+* :ref:`HTTP internal redirects <arch_overview_internal_redirects>`
```

Comment:
Is this an extension point?

Reply:
This was based off of pr #10908. 

In https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/route/v3/route_components.proto#L1722  there's a `core.v3.TypedExtensionConfig`. I linked `arch_overview_internal_redirects` as in the PR where the change was made `http_connection_management.rst` was modified primarily in that section it seems to accommodate.

I will revert here, but if you think this makes sense I'm happy to add this in another PR.

Reply:
I see, OK. I guess I would mildly prefer that we ref link directly to the TypedExtensionConfig field to make it more obvious along with any arch docs. In any case, I don't want to hold up your PR, but any follow ups to improve the docs would be appreciated!

Reply:
Added a separate PR for this: #12620